BlackMarket/app/Components/SecureTextFieldWithReveal.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  SecureTextFieldWithReveal.swift
+//  BlackMarket
+//
+//  Created by Tarek Radovan on 07/12/2022.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct SecureTextFieldDomain: ReducerProtocol {
```

Comment:
The BMButton is no longer needed.. just `BMTextField` has the prefix BM because is the custom textfield of this app ... the `SecureTextfieldWithReveal` is not carrying with the BM prefix because is just a part of `BMTextfield`